resources/js/components/Product/AddToCart.vue

```diff
@@ -65,13 +66,19 @@
             },
 
             async add() {
+                return this.addQty(this.qty);
+            },
+
+            async addQty(count) {
```

Comment:
```suggestion
            async add(count = 1) {
```

Reply:
This wouldn't solve the backwards compatibility issue because in some cases, `@click="add"` is used, causing the `$event` to be passed as the `count` parameter.